e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -13,9 +13,9 @@ param(
     [Parameter(Mandatory)]
     [string] $GroupCertificatePassword,
 
-    # Set this to true on the first execution to get everything installed in powershell. Does not need to be run everytime.
+    # Specify this on the first execution to get everything installed in powershell. It does not need to be run every time.
     [Parameter()]
-    [bool] $InstallDependencies = $true,
+    [switch] $InstallDependencies,
```

Comment:
I almost never need to include this. If I forget, it installs by default and I cancel and rerun. I have to remember to include it with a second parameter of `$false`.

Instead, use the switch type to turn it on when needed.